index.js

```diff
@@ -78,7 +78,7 @@ const destroyCircular = ({
 			from: value,
 			seen: [...seen],
 
-			to_: isErrorLike(value) ? new Error() : undefined,
+			to_: isMinimumViableSerializedError(value) ? new Error() : undefined,
```

Comment:
This fixes the inconsistent expectation between the minimum viable error at the top level and for nested errors.

- `{}` was deserialized as an error
- `{cause: {name: 'Error', message: 'foo'}` was serialized as an error, but `cause` wasn't

After #73 this check should also be changed in the new code there.